src/main/resources/logback.xml

```diff
@@ -11,41 +10,33 @@
         </layout>
     </appender>
 
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <File>${log-path}/beamLog.out</File>
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+            </Pattern>
+        </layout>
+    </appender>
+
     <root level="info">
         <appender-ref ref="STDOUT"/>
+        <appender-ref ref="FILE"/>
     </root>
 
-    <logger name="com.conveyal" level="error" additivity="false">
-        <appender-ref ref="STDOUT"/>
-    </logger>
+    <logger name="com.conveyal" level="error" />
 
     <!--    MATSIM -->
-    <logger name="org.matsim" level="ERROR">
-        <appender-ref ref="STDOUT"/>
-    </logger>
-    <Logger name="org.matsim.core.controler.corelisteners.DumpDataAtEndImpl" level="OFF">
-        <appender-ref ref="STDOUT"/>
-    </Logger>
-    <Logger name="org.matsim.core.events" level="WARN">
```

Comment:
Did you mean to lose `org.matsim.core.events`?

Reply:
We're using the fork of matsim with slight modifications. When there will be error during event's processing, now it will be logged under `ERROR` level: https://github.com/wrashid/matsim/blob/0.10.x/matsim/src/main/java/org/matsim/core/events/EventsManagerImpl.java#L234. We still have `ERROR` log level for MATSim: `<logger name="org.matsim" level="ERROR" />`